sdk/metricsadvisor/azure-ai-metricsadvisor/README.md

```diff
@@ -10,7 +10,8 @@ Metrics Advisor is a scalable real-time time series monitoring, alerting, and ro
 
 ## _Disclaimer_
 
-_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
```

Comment:
Sorry if I was unclear here - let's keep the line about 2.7 and remove the line about 3.7. :)